mypy/types.py

```diff
@@ -1162,23 +1162,45 @@ class TypeType(Type):
     # a generic class instance, a union, Any, a type variable...
     item = None  # type: Type
 
+    # this class should not be created directly
+    # use make method which may return either TypeType or UnionType
+    # this is to ensure Type[Union[A, B]] is always represented as Union[Type[A], Type[B]]
     def __init__(self, item: Type, *, line: int = -1, column: int = -1) -> None:
```

Comment:
But how would we know (statically) whether to call `make_normalized` or normal constructor? In some cases, we don't if the type is `Union` or not.